app/src/main/java/com/android/voyageur/ui/overview/AddTrip.kt

```diff
@@ -192,7 +198,7 @@ fun AddTripScreen(
                     }
                   },
                   enabled = name.isNotBlank() && startDate.isNotBlank() && endDate.isNotBlank(),
```

Comment:
I think you should also add a Toast requesting the user to enter a name for the trip in case they leave it empty (like in the bootcamp).